OpenRA.Mods.Common/Widgets/ScrollItemWidget.cs

```diff
@@ -57,6 +57,7 @@ public static ScrollItemWidget Setup(ScrollItemWidget template, Func<bool> isSel
 			w.IsVisible = () => true;
 			w.IsSelected = isSelected;
 			w.OnClick = onClick;
+			w.ItemKey = Convert.ToString(Game.CosmeticRandom.Next());
```

Comment:
ItemKey is not a hotkey but an identifying text for the button. Map chooser window uses map UID as ItemKey for example.